libs/methodologies/bold/rule-processors/mass-id/src/mass-definition/mass-definition.processor.ts

```diff
@@ -7,25 +7,30 @@ import {
   DocumentCategory,
   DocumentType,
   MassSubtype,
-  MeasurementUnit,
+  NewMeasurementUnit,
 } from '@carrot-fndn/shared/methodologies/bold/types';
 import { RuleOutputStatus } from '@carrot-fndn/shared/rule/types';
 
-import { CheckOrganicMassCriteriaProcessorErrors } from './check-organic-mass-criteria.errors';
+import { MassDefinitionProcessorErrors } from './mass-definition.errors';
 
-export class CheckOrganicMassCriteriaProcessor extends ParentDocumentRuleProcessor<Document> {
-  protected readonly processorErrors =
-    new CheckOrganicMassCriteriaProcessorErrors();
+const ALLOWED_SUBTYPES: string[] = Object.values(MassSubtype);
+
+export class MassDefinitionProcessor extends ParentDocumentRuleProcessor<Document> {
+  protected readonly processorErrors = new MassDefinitionProcessorErrors();
 
   private get RESULT_COMMENT() {
     return {
-      APPROVED: 'The Organic MassID criteria is met.',
-      CATEGORY_NOT_MATCHING: `Category does not match with ${DocumentCategory.MASS_ID}`,
-      CURRENT_VALUE_NOT_MATCHING:
-        'The MassID current value is not greater than 0',
-      MEASUREMENT_UNIT_NOT_MATCHING: `Measurement unit does not match with ${MeasurementUnit.KG}`,
-      SUBTYPE_NOT_MATCHING: `Subtype does not match with any of the allowed subtypes`,
-      TYPE_NOT_MATCHING: 'The Organic MassID criteria is not met.',
+      APPROVED:
+        'Compliant mass definition: category, type, subtype, value and measurement unit.',
```

Comment:
I think that this phrase should have a verb, here is a suggestion
```suggestion
        'The mass definition is compliant: category, type, subtype, value, and measurement unit are valid. ',
```

Reply:
https://github.com/carrot-foundation/methodology-rules/pull/147/commits/ffe3c3ac93873f0965e0b42a55e33c1067bcfd51